doc/whatsnew/2.0.rst

```diff
@@ -37,6 +37,17 @@ New checkers
     class A:    # better
         pass
 
+* A new check was added, ``misplaced-format-function``.
+
+  This message is emitted when pylint detects that a format function is called on non str object.
+  This can occur due to wrong placement of closing bracket, e.g
+
+  .. code-block:: python
```

Comment:
This code block isn't showing up. I think it's because you don't have an empty line after the directive. You can test this on github view.

Reply:
thanks @brycepg for finding this issue, code block not visible for infact many checkers implemented by me.
However I have moved documentation of this checker from doc/whatsnew/2.0.rst to doc/whatsnew/2.1.rst as 2.0 is already released.